app/views/admin/educators/authorization.html.erb

```diff
@@ -1,100 +1,108 @@
+        <% educators.map do |educator| %>
+          <tr>
+            <td><%= educator.email %></td>
+            <td><%= link_to educator.full_name, "/educators/view/#{educator.id}" %></td>
+            <td><%= educator.can_set_districtwide_access %></td>
+            <td><%= educator.admin %></td>
+            <td><%= educator.can_view_restricted_notes %></td>
+            <td><%= educator.districtwide_access %></td>
+            <td><%= educator.school.try(:name) || "N/A" %></td>
         <% end %>
       </tbody>
     </table>
-</div>
+  </div>
+
+  <h2>Educator homepages</h2>
+  <div style="padding: 20px">
+    <table style="margin-top: 40px; text-align: left; border-collapse: collapse;">
+        <thead>
+          <tr style="vertical-align: top;">
+            <th style="padding: 5px; background: #ccc;">Last login<br/>(days ago)</th>
+            <th style="padding: 5px; background: #ccc;">Logins<br/>(all time)</th>
+            <th style="padding: 5px; background: #ccc;">Email</th>
+            <th style="padding: 5px; background: #ccc;">Name</th>
+            <th style="padding: 5px; background: #ccc;">Masquerade</th>
+            <th style="padding: 5px; background: #ccc;">School</th>
+            <th style="padding: 5px; background: #ccc;">Links</th>
+          </tr>
+        </thead>
+        <tbody>
+          <%
+            @navbar_links_map = {}
+            sorted_educators = @all_educators.sort_by do |educator|
+              navbar_links = PathsForEducator.new(educator).navbar_links
+              @navbar_links_map[educator.id] = navbar_links
+              navbar_links.keys
+            end
+          %>
+          <% sorted_educators.map do |educator| %>
+            <%
+              days_since_last_sign_in = if educator.last_sign_in_at.nil?
+                0
+              else
+                ((Time.now - educator.last_sign_in_at) / 1.day).round
+              end
+              color = if days_since_last_sign_in == 0
+                'white'
+              elsif days_since_last_sign_in < 7
+                '#006d2c'
+              elsif days_since_last_sign_in < 30
+                '#31a354'
+              elsif days_since_last_sign_in < 90
+                '#a1d99b'
+              elsif days_since_last_sign_in < 365
+                '#c7e9c0'
+              else
+                '#f7fcf5'
+              end
+            %>
+            <tr style="vertical-align: top;">
+              <td style="padding: 5px;">
+                <span style="display: inline-block; width: 2em; text-align: right; color: white; padding: 5px; background: <%= color %>;">
+                  <%= days_since_last_sign_in %>
+                </span>
+              </td>
+              <td style="padding: 5px;"><%= educator.sign_in_count %></td>
+              <td style="padding: 5px;"><%= educator.email %></td>
+              <td style="padding: 5px;"><%= link_to educator.full_name, "/educators/view/#{educator.id}" %></td>
+              <td>
```

Comment:
This is the meaningful change; the rest is whitespace noise from adding a wrapping div that does some styling.